base/sparse/sparsematrix.jl

```diff
@@ -516,6 +508,26 @@ for op in (:-, :abs, :abs2, :log1p, :expm1)
     end
 end
 
+function abs{Tv<:Complex,Ti}(A::SparseMatrixCSC{Tv,Ti})
+    T = Tv.parameters[1]
+    (Tv.parameters[1] <: Integer) && (T = Float64)
```

Comment:
should `abs(A::SparseMatrixCSC{Complex{BigInt},Ti})` return `SparseMatrixCSC{BigFloat,Ti}` ? I think that would be consistent with the scalar case